api/api.go

```diff
@@ -254,5 +254,8 @@ func NewServerRoutes(s Server) *http.ServeMux {
 	sqlh := NewSQLHandlers(s.Instance, cfg.API.ReadOnly)
 	m.Handle("/sql", s.middleware(sqlh.QueryHandler("/sql")))
 
+	wh := NewWebuiHandlers(s.Instance, !cfg.API.DisableWebui)
+	m.Handle("/webui", s.middleware(wh.WebuiHandler))
```

Comment:
This can be refactored to be totally stateless:

```suggestion
	if !cfg.API.DisableWebui {
	  m.Handle("/webui", s.middleware(WebuiHandler))
	}
```

```go
// webui.go
package api

func WebuiHandler(w http.ResponseWriter, r *http.Request) {
  template :=  `
 <!doctype html><html><head><link href="https://storage.googleapis.com/qri-static/bundle.css" rel="stylesheet"></head><head><title>Qri Remote Webapp</title><meta charset="utf-8"><meta http-equiv="Content-Security-Policy" content="script-src 'self' https://storage.googleapis.com http://localhost:2503 https://qri.io https://qri.cloud"><body><div class="titlebar"></div><div id="root"></div><script src="https://storage.googleapis.com/qri-static/bundle.js"></script></body></head></html>
 `

  w.Write([]byte(template))
}
```

putting the template in the handler means memory is only allocated during a request